modules/Ecommerce/DealDay/Database/Migrations/2025_11_16_120100_add_date_offer_to_deal_days_table.php

```diff
@@ -6,7 +6,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
-return new class extends Migration
+return new class 2025_11_16_120100_add_date_offer_to_deal_days_table Migration
```

Comment:
The class name syntax is incorrect. It should be `extends Migration` without the timestamp prefix. The timestamp is already part of the filename.

```suggestion
return new class extends Migration
```